generators/server/templates/pom.xml.ejs

```diff
@@ -206,10 +206,19 @@
             <groupId>com.fasterxml.jackson.datatype</groupId>
             <artifactId>jackson-datatype-jsr310</artifactId>
         </dependency>
+<%_ if (javaVersion === '1.8') { _%>
         <dependency>
             <groupId>com.fasterxml.jackson.module</groupId>
             <artifactId>jackson-module-afterburner</artifactId>
         </dependency>
+<%_ } _%>
+<%_ if (javaVersion === '11' && useJacksonBlackBird === 'Y') { _%>
+        <dependency>
+            <groupId>org.jdbi.jackson-contrib</groupId>
+            <artifactId>jackson-module-blackbird</artifactId>
+            <version>0.0.1</version>
```

Comment:
Can you please create a property for the version: jackson-module-blackbird.version?

Reply:
@DanielFran : Thanks; done 😄